nrfu_tests/test_definition.yaml

```
@@ -0,0 +1,12 @@
+- name: nrfu_tests
+  testcases:
+  - name: test_base_services_tacacs
+    description: Testcase for verification of TACACS servers details.
+    test_id: NRFU1.4
+    show_cmd: show tacacs
+    expected_output: null # Forming expected output in test case file.
+    test_criteria: TACACS servers should not have errors, timeouts, failures or disconnects. # Setting test case’s pass/fail criteria for reporting.
+    report_style: modern ## Setting report_style as modern If report type is set to modern, then it will update steps, assumptions, and external systems in the docx report.
```

```suggestion
    report_style: modern
```
Remove the inline comments. I think it's not needed here.